draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -882,6 +882,61 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 TBD
 
+## Media Type Registration
+
+This section registers the "application/sdjwt-issuance" media type [@RFC2046] in
+the "Media Types" registry [@IANA.MediaTypes] in the manner described
+in [@RFC6838], which can be used to indicate that the content is a Combined Format for Issuance.
+This section also registers the "application/sdjwt-presentation" media type
+in the "Media Types" registry, which can be used to indicate that
+the content is a Combined Format for Presentation.
+
+### application/sdjwt-issuance
+
+*  Type name: application
+*  Subtype name: sdjwt-issuance
+*  Required parameters: n/a
+*  Optional parameters: n/a
+*  Encoding considerations: 8bit; application/sdjwt-issuance values are encoded as a series of base64url-encoded values, each separated from the next by a single tilde ('~') character.
```

Comment:
I don't know how specific the  Encoding considerations needs to be but the dot character also is a separator in the JWT part.

Reply:
```suggestion
*  Encoding considerations: 8bit; application/sd-jwt-issuance values are encoded such that the first three base64url-encoded values are separated from the previous one by a single period ('.') character, and the following series of base64url-encoded values are each separated from the next by a single tilde ('~') character.
```

Reply:
I tried.

Reply:
```suggestion
*  Encoding considerations: 8bit; application/sdjwt-issuance values are encoded as a series of strings separated from the next by a single tilde ('~') character; the first string consists of dot-separated base64url-encoded values, all further strings are base64url-encoded values.
```

Reply:
```suggestion
*  Encoding considerations: 8bit; application/sdj-wt-issuance values are a series of base64url-encoded values (some of which may be the empty string) separated by period ('.') or tilde ('~') characters.
```

This tries to simplify it and make it less specific.